core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -119,24 +119,26 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.getBindableQuestion(
           stateName).setBridgeHtml(newHtml);
       },
-      addNewQuestion: function() {
-        // This effectively adds a new multiple-choice interaction to the
-        // latest state in the chain.
-        var lastStateName = (
-          data.questionList.isEmpty() ?
-          SimpleEditorShimService.getInitStateName() :
-          data.questionList.getLastQuestion().getDestinationStateName());
+      addNewQuestion: function(type) {
+        if(type == 'multiple'){
+          // This effectively adds a new multiple-choice interaction to the
+          // latest state in the chain.
```

Comment:
Yes, let's start by adding a multiple-choice card directly. I think it makes sense to keep the "add" button at the bottom pretty simple for the time being, and it also seems better to avoid cards with "null" interactions. We can revisit later once more of the overall flow is built out, and we could always choose a different interaction as the "default" later.